examples/horizontal_convection.jl

```diff
@@ -0,0 +1,356 @@
+# and the buoyancy dissipation, ``\chi = \kappa |\boldsymbol{\nabla}b|^2``. Note that 
+# `ComputedField`s take "AbstractOperations"on `Field`s as input:
+
+u, v, w = model.velocities # unpack velocity `Field`s
+b = model.tracers.b        # unpack buoyancy `Field`
+
+## total flow speed
+speed = ComputedField(sqrt(u^2 + w^2))
+
+## y-component of vorticity
+ζ = ComputedField(∂z(u) - ∂x(w))
+
+## buoyancy dissipation
+χ_op = @at (Center, Center, Center) κ * (∂x(b)^2 + ∂z(b)^2)
+χ = ComputedField(χ_op)
+
+outputs = (s = speed, b = b, ζ = ζ, χ = χ)
+nothing # hide
+
+# We create a `JLD2OutputWriter` that saves the speed, the vorticity, and the buoyancy dissipation.
+# We then add the `JLD2OutputWriter` to the `simulation`.
+
+simulation.output_writers[:fields] = JLD2OutputWriter(model, outputs,
+                                                  field_slicer = nothing,
+                                                      schedule = TimeInterval(0.1),
+                                                        prefix = "horizontal_convection",
+                                                         force = true)
+nothing # hide
+
+# Ready to press the big red button:
+
+run!(simulation)
+
+# ## Visualizing horizontal convection
+#
+# We animate the results by opening the JLD2 file, extracting data for the iterations we ended
+# up saving at, and ploting the saved fields. We prepare for animating the flow by creating 
+# coordinate arrays, opening the file, building a vector of the iterations that we saved data at,
+# and defining a function for computing colorbar limits:
+
+using JLD2, Plots
+
+## Coordinate arrays
+xs, ys, zs = nodes(speed)
+xb, yb, zb = nodes(b)
+xζ, yζ, zζ = nodes(ζ)
+xχ, yχ, zχ = nodes(χ)
+
+## Open the file with our data
+file = jldopen(simulation.output_writers[:fields].filepath)
+
+## Extract a vector of iterations
+iterations = parse.(Int, keys(file["timeseries/t"]))
+
+# The utilities below come handy for calculating nice contour intervals:
+
+function nice_levels(c, clim, nlevels=41)
+    levels = range(0, stop=clim, length=nlevels)
+    cmax = maximum(abs, c)
+    clim < cmax && (levels = vcat(levels, [cmax]))
+    return levels
+end
+
+function nice_divergent_levels(c, clim, nlevels=41)
+    levels = range(-clim, stop=clim, length=nlevels)
+    cmax = maximum(abs, c)
+    clim < cmax && (levels = vcat([-cmax], levels, [cmax]))
+    return levels
+end
+
+nothing # hide
+
+# Now we're ready to animate.
+
+@info "Making an animation from saved data..."
+
+anim = @animate for (i, iter) in enumerate(iterations)
+
+    ## Load fields from file
+    t = file["timeseries/t/$iter"]
+    s_snapshot = file["timeseries/s/$iter"][3:end-3, 1, 4:end-3]
+    b_snapshot = file["timeseries/b/$iter"][4:end-3, 1, 4:end-3]
+    ζ_snapshot = file["timeseries/ζ/$iter"][4:end-3, 1, 4:end-3]
+    χ_snapshot = file["timeseries/χ/$iter"][4:end-3, 1, 4:end-3]
```

Comment:
To be able to use the `FieldTimeSeries` to post-process output I had to save the halo points. But for plotting I don't want the halo points. What's the best way forward? Obviously hardcoding things like that is clumsy...


Should I load each field with `FieldTimeSeries` and use `interior()` on that? (That won't work at the moment but we can write a method for `interior` on `FieldTimeSeries`...) Or anybody has a better suggestion?

cc: @glwagner, @ali-ramadhan

Reply:
Use field functionality for plotting, like `interior`.

Reply:
`interior` definitely works with `FieldTimeSeries`, et

```julia
s = interior(s_timeseries[1])
```